jaeger-micrometer/README.md

```diff
@@ -0,0 +1,159 @@
+# Micrometer integration for Jaeger internal metrics
+
+This module brings a Micrometer integration to the internal Jaeger metrics.
+
+## Configuring
+
+To configure the Micrometer integration, a custom Jaeger Tracer instance has to be used:
+
+```java
+MicrometerMetricsFactory metricsReporter = new MicrometerMetricsFactory();
+Configuration configuration = new Configuration("myServiceName");
+Tracer tracer = configuration
+    .getTracerBuilder()
+    .withMetrics(new com.uber.jaeger.metrics.Metrics(metricsReporter))
```

Comment:
nit: I think there is a direct with method to set metrics reporter

Reply:
Was wondering about that - the `com.uber.jaeger.metrics.Metrics` class can not just be an internal impl detail, and only provide the `MetricsFactory`.